src/GovUk.Education.ExploreEducationStatistics.Content.Api/Services/FileStorageService.cs

```diff
@@ -1,22 +1,22 @@
 using System.Threading.Tasks;
-using GovUk.Education.ExploreEducationStatistics.Common.Services;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces;
 using GovUk.Education.ExploreEducationStatistics.Content.Api.Services.Interfaces;
 using static GovUk.Education.ExploreEducationStatistics.Common.BlobContainerNames;
 
 namespace GovUk.Education.ExploreEducationStatistics.Content.Api.Services
 {
     public class FileStorageService : IFileStorageService
     {
-        private readonly string _storageConnectionString;
+        private readonly IBlobStorageService _blobStorageService;
 
-        public FileStorageService(string connectionString)
+        public FileStorageService(IBlobStorageService blobStorageService)
         {
-            _storageConnectionString = connectionString;
+            _blobStorageService = blobStorageService;
         }
 
         public async Task<string> DownloadTextAsync(string blobName)
         {
-            return await FileStorageUtils.DownloadTextAsync(_storageConnectionString, PublicContentContainerName, blobName);
+            return await _blobStorageService.DownloadBlobText(PublicContentContainerName, blobName);
```

Comment:
Do you think there's much point in this FileStorageService existing at the moment? I've noticed other services are ok to use BlobStorageService directly like the DataArchiveService.

Reply:
In this instance nope! I'll get rid of it.